models/articles.model.js

```diff
@@ -28,3 +28,13 @@ exports.fetchArticles = () => {
       else return articles;
     });
 };
+
+exports.checkArticleExists = (article_id) => {
+  return db
+    .query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then(({ rows: articles }) => {
+      if (!articles.length) {
+        return Promise.reject({ status: 404, msg: "Not Found" });
```

Comment:
Custom errors are really great opportunities to be quite specific about what the problem is to help the user know what went wrong.